### PR TITLE
yapf_2019_08_09

### PR DESCRIPTION
**Pull request by pgunn**
Ran yapf on files that haven't changed in the last 15 days or so.

### CODE REVIEW

File: caiman/tests/comparison_humans_online.py [hunk 1]
```diff
@@ -101,26 +101,19 @@
...
-params_movie[1] = {
-    'folder_name': 'N.01.01/',
-    'ds_factor': 1,
-    'fr': 8,
-    'gnb': 1,
-    'decay_time': 1.4,
-    'gSig': [6, 6]
-}
+                                       # % neurofinder.01.01
+params_movie[1] = {'folder_name': 'N.01.01/', 'ds_factor': 1, 'fr': 8, 'gnb': 1, 'decay_time': 1.4, 'gSig': [6, 6]}
```
Comment (epnev) [inline caiman/tests/comparison_humans_online.py]: @pgunn 
kind of weird it chose to turn this into a single line but not the rest of the entries.

File: caiman/tests/test_temporal.py [hunk 2]
```diff
@@ -10,12 +10,7 @@ def test_make_G_matrix():
...
     G = cnmf.temporal.make_G_matrix(T, g)
     G = G.todense()
-    true_G = np.matrix(
-        [[1., 0., 0., 0., 0., 0.],
-         [-1., 1., 0., 0., 0., 0.],
-         [-2., -1., 1., 0., 0., 0.],
-         [-3., -2., -1., 1., 0., 0.],
-         [0., -3., -2., -1., 1., 0.],
-         [0., 0., -3., -2., -1., 1.]])
+    true_G = np.matrix([[1., 0., 0., 0., 0., 0.], [-1., 1., 0., 0., 0., 0.], [-2., -1., 1., 0., 0., 0.],
```
Comment (epnev) [inline caiman/tests/test_temporal.py]: older is easier to read here given it shows the matrix.

File: caiman/tests/comparison_humans_online.py (hunk 1 above)
Comment (pgunn) [reply caiman/tests/comparison_humans_online.py]: I think the lack of a comment for this structure gave it some more flexibility. I should make how it does these consistent.

File: caiman/tests/test_temporal.py (hunk 2 above)
Comment (pgunn) [reply caiman/tests/test_temporal.py]: ok, will revert and prevent it from doing this again.

File: caiman/tests/comparison_humans_online.py (hunk 1 above)
Comment (epnev) [reply caiman/tests/comparison_humans_online.py]: really minor points, no need to spend time on this.

**Comment by epnev** [comment]
@pgunn I'll process first #612 and #619 and then come to this to avoid any potential conflicts. If you want to yapf a bit more do it directly here, although it might be prudent to wait until the code stabilizes reaches a steady state.

**Comment by pgunn** [comment]
Whatever timing is easiest to do ; I can easily reconstruct the diff from scratch using just the tool and lists of things where we want to keep the existing formatting. If it makes it easier to merge I can redo this focused around files that haven't changed in a month rather than 10-ish days.

**Pull request merged**